docs/api.md

```diff
@@ -10,4 +10,24 @@
 
 ::: geff.validate
 
-::: geff.GeffMetadata
\ No newline at end of file
+## Metadata
+
+::: geff.GeffMetadata
+
+::: geff.metadata_schema.Axis
+
+::: geff.metadata_schema.axes_from_lists
+
+## Units
+
+TODO: provide valid units in the docs somehow
+
+::: geff.units.validate_axis_type
+
+::: geff.units.validate_space_unit
+
+::: geff.units.validate_time_unit
```

Comment:
```suggestion
::: geff.units.validate_axis_type

::: geff.units.VALID_AXIS_TYPES

::: geff.units.validate_space_unit

::: geff.units.VALID_SPACE_UNITS

::: geff.units.validate_time_unit

::: geff.units.VALID_TIME_UNITS
```

Reply:
Make the valid units/types show up in the api docs alongside the functions that are doing the validation